tests/cpp_tests/test_stream.cpp

```diff
@@ -320,7 +320,7 @@ TEST(Stream, PushSparseRowsWithMetadata) {
   TestUtils::CreateRandomSparseData(nrows, ncols, nclasses, sparse_percent, &indptr, &indices, &vals, &labels, &weights, &init_scores, &groups);
 
   const std::vector<int32_t> batch_counts = { 1, nrows / 100, nrows / 10, nrows };
-  const std::vector<int8_t> creation_types = { 0, 1 };
+  const std::vector<int8_t> creation_types = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0 };
```

Comment:
@svotaw 
Why there are only zeros here now and no any ones?
Previously `creation_types = { 0, 1 };` made sense to me, but now I'm very confused by such test parametrization.

Reply:
@StrikerRUS  oops, yes need to revert this.  It was a quick and dirty experiment just to force many repeats.  Some of the early testing failures were only sporadic, so I added this to give me better confidence on no failures.  I will fix.

I had planned to do a last pass over the PR to look for things like this, but it got checked in.  Not used to someone else being in control of the checkin. :)

Reply:
No problem, thanks a lot for your awesome work on LightGBM!

Luckily, we need to revert just one line of the code. 🙂